### PR TITLE
docs: proofs

### PR DESCRIPTION
**Pull request by MosCD3**
Initial documentation for dealing with Proofs

### CODE REVIEW

File: docs/getting-started/6-proofs.md [hunk 1]
```diff
@@ -1,3 +1,130 @@
...
+As mentioned in the previous documentation ([Credentials](5-credentials.md)), after receiving a credential and saving it to your wallet, you will need to show it to a verifier who will verify the authenticity of this credential and that the credential assertions are not tampered with.
+
+In VC proofs, we have two involved parties:
+
+- Holder (prover)
+- Verifier
+
+The process for proving your VC starts by a verifier to request a presentation from a prover, and for the prover to respond by presenting a proof to the verifier or the prover to send a presentation proposal to the verifier.
+
+## Method 1 - Prover (holder) responds to presentation proposal from the verifier
```
Comment (TimoGlastra) [inline docs/getting-started/6-proofs.md:12]:
```suggestion
## Method 1 - Prover (holder) responds to presentation request from the verifier
```

File: docs/getting-started/6-proofs.md [hunk 2]
```diff
@@ -1,3 +1,130 @@
...
+```
+
+- `filterByPresentationPreview`: Whether to filter the retrieved credentials using the presentation preview. This configuration will only have effect if a presentation proposal message is available containing a presentation preview..
+
+Make sure to add the event listener to the agent after initializing the wallet
+
+```ts
+agent.events.on<ProofStateChangedEvent>(ProofEventTypes.ProofStateChanged, (event) => {
+  handleProofStateChange(event)
+})
```
Comment (TimoGlastra) [inline docs/getting-started/6-proofs.md:54]:
```suggestion
agent.events.on<ProofStateChangedEvent>(ProofEventTypes.ProofStateChanged, handleProofStateChange)
```

File: docs/getting-started/6-proofs.md [hunk 3]
```diff
@@ -1,3 +1,130 @@
...
+  message += `${attr.name}\n`
+})
+
+message += `Accept proof request?`
+```
+
+By sending the response to the verifier, the verifier will go through the process of verifying the VC and respond with an ack message.
+To give some context to the user you can add the following code to the Proof event handler
+
+```ts
```
Comment (TimoGlastra) [inline docs/getting-started/6-proofs.md:103]: I feel like all the different code blocks make it quite hard to understand how they fit together. I think first making a full e2e flow in a single code block that is as simple as possible (so no `message = `, `console.log` etc...) and then add different sections going into more detail on specific topics such getting the requested values, auto selecting credentials, manually selecting credentials, ...

**Review by TimoGlastra — CHANGES_REQUESTED** [review]
Thanks again for the work @MosCD3.

File: docs/getting-started/6-proofs.md (hunk 3 above)
Comment (MosCD3) [reply docs/getting-started/6-proofs.md:101]: sounds good

Comment (MosCD3) [reply docs/getting-started/6-proofs.md:120]: done

**Comment by MosCD3** [comment]
@TimoGlastra Not sure what brought all those other ci changes to this commit, any idea what to do?

**Comment by MosCD3** [comment]
@TimoGlastra are we done with that? 
let me know if you need more changes

**Review by TimoGlastra — APPROVED** [review]
👍

**Pull request merged**